testing/percona-xtrabackup/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Contributor: Peter Szalatnay <theotherland@gmail.com>
+# Maintainer: Peter Szalatnay <theotherland@gmail.com>
+pkgname=percona-xtrabackup
+pkgver=2.4.6
+pkgrel=0
+pkgdesc='Percona XtraBackup is a MySQL non-blocking backup solution for InnoDB and XtraDB data'
+arch=all
+url='http://www.percona.com/software/percona-xtrabackup/'
+license='GPL'
+makedepends='bash bison cmake ncurses-dev libaio-dev libgcrypt-dev zlib-dev curl-dev libev-dev vim'
```

Comment:
It is a dependency it needs vim-common on debian and it does not build without it.